src/main/java/com/craftingguide/exporter/extensions/minecraft/MinecraftExtension.java

```diff
@@ -16,6 +16,7 @@ public void register(Registry registry) {
 
         registry.registerWorker("minecraft.FurnaceRecipeGatherer", Priority.LOWEST);
         registry.registerWorker("minecraft.PotionRecipeGatherer", Priority.LOWEST);
+        registry.registerWorker("minecraft.OtherRecipeGatherer", Priority.LOWEST);
```

Comment:
Let's make this an NEI extension instead.  The `minecraft` package is specifically for code which has to do with the vanilla game.  Every other mod (including Crafting Guide itself!) is placed into it's own extension package.